integration_tests/src/main/python/orc_write_test.py

```diff
@@ -64,13 +68,14 @@
     ArrayGen(StructGen([['child0', byte_gen], ['child1', string_gen], ['child2', float_gen]]))]
 
 orc_write_basic_map_gens = [simple_string_to_string_map_gen] + [MapGen(f(nullable=False), f()) for f in [
```

Comment:
Do we need the boolean issue comment here as well so it gets restored when fixed?